docs/config.md

```diff
@@ -1,211 +1,466 @@
-   audio using Jitsi's live streaming feature. This option is experimental and
-   may be removed at any time without notice.
-1. `voip`: Behaviour related to calls
-   1. `obeyAssertedIdentity`: If set, MSC3086 asserted identity messages sent
-      on VoIP calls will cause the call to appear in the room corresponding to the
-      asserted identity. This *must* only be set in trusted environments.
-1. `posthog`: [Posthog](https://posthog.com/) integration config. If not set, Posthog analytics are disabled.
-   1. `projectApiKey`: The Posthog project API key
-   2. `apiHost`: The Posthog API host
-1. `sentry`: [Sentry](https://sentry.io/) configuration for rageshake data being sent to sentry.
-   1. `dsn`: the Sentry [DSN](https://docs.sentry.io/product/sentry-basics/dsn-explainer/)
-   2. `environment`: (optional) The [Environment](https://docs.sentry.io/product/sentry-basics/environments/) to pass to sentry
-1. `map_style_url`: Map tile server style URL for location sharing. e.g.
-   'https://api.maptiler.com/maps/streets/style.json?key=YOUR_KEY_GOES_HERE'
-  This setting is ignored if your homeserver provides
-  `/.well-known/matrix/client` in its well-known location, and the JSON file
-  at that location has a key `m.tile_server` (or the unstable version
-  `org.matrix.msc3488.tile_server`). In this case, the configuration found in
-  the well-known location is used instead.
-1. `analyticsOwner`: The entity that analytics data is being sent to. Used in copy
-   when explaining to the user where data is being sent. If not set, defaults to `brand`.
-1. `defaultDeviceDisplayName`: The default device display name to use for new logins
-   and registrations. If not set then a calculated version will be used.
-1. `custom_translations_url`: An optional URL to allow overriding of translatable strings.
-   The JSON file must be in a format of `{"affected string": {"languageCode": "new string"}}`.
-   See https://github.com/matrix-org/matrix-react-sdk/pull/7886 for details.
-
-Note that `index.html` also has an og:image meta tag that is set to an image
-hosted on riot.im. This is the image used if links to your copy of Element
-appear in some websites like Facebook, and indeed Element itself. This has to be
-static in the HTML and an absolute URL (and HTTP rather than HTTPS), so it's
-not possible for this to be an option in config.json. If you'd like to change
-it, you can build Element, but run
-`RIOT_OG_IMAGE_URL="http://example.com/logo.png" yarn build`.
-Alternatively, you can edit the `og:image` meta tag in `index.html` directly
-each time you download a new version of Element.
-
-Identity servers
-================
+# Configuration
+
+You can configure the app by copying `config.sample.json` to `config.json` and customising it. The file is in JSON
+format and can often result in syntax errors when editing it - visit [#element-web:matrix.org](https://matrix.to/#/#element-web:matrix.org)
```

Comment:
> The file is in JSON
format and can often result in syntax errors when editing it

Would probably suggest removing that. If we're worried about syntax issues here, why not include configs in the list of files watched by ESLint?

Reply:
I'm just not convinced we need to warn people that JSON can be error-prone to write.
I agree, a clearer error message when deploying an incorrect config would have way bigger impact